sql/test_data.sql

```diff
@@ -83,8 +83,17 @@ INSERT INTO position_holders(pos_id, user_id) VALUES
     (4, 2),
     (5, 4);
 
+
 INSERT INTO position_relations(pos_id_from, pos_id_to) VALUES
     (4, 5);
 
 INSERT INTO rooms(room_id, location, title, description) VALUES
     (1, 'SAC 23', 'ASCIT Screening Room', 'A room for watching DVDs and videos');
+
+INSERT INTO permissions(permission_id, permission_name) VALUES
```

Comment:
Shouldn't have to specify `permission_id` here, since it auto-increments

Reply:
Just to make it clear when comparing the tests to the test data